core/modules/user/user.actions.inc

```diff
@@ -13,6 +13,18 @@
  * @ingroup actions
  */
 function user_block_user_action(User $account, &$context) {
+  global $user;
+
+  // Don't block User 1, or the user performing the action.
+  if ($account->uid === 1) {
```

Comment:
`$account->uid` is actually a string, not an integer

Reply:
Fixed, thanks!